Src/FluentAssertions.Net40/Xml/AndWhichConstraint.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace FluentAssertions.Xml
+{
+    public class AndWhichConstraint<T>
```

Comment:
I don't need it. It's a stupid mistake (Ctrl+. going wild). I'll remove it. Sorry for the mess.